automatic/git.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,32 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$is64bit = Get-ProcessorBits 64
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$filePath32 = "$toolsDir\PortableGit-2.11.0-32-bit.7z.exe"
+$filePath64 = "$toolsDir\PortableGit-2.11.0-64-bit.7z.exe"
+$filePath = if ($is64bit -and $env:chocolateyForceX86 -eq $false) { $filePath64 } else { $filePath32 }
```

Comment:
Its the best to have something like this:

```powershell
$embedded_path = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
         Write-Host "Installing 64 bit version"; gi "$toolsDir\*-64-*.exe"
} else { Write-Host "Installing 32 bit version"; gi "$toolsDir\*-32-*.exe" }
```

Write-Host is good to have because choco doesn't show anything here unlike with online version.